CSharp/oib-validation.cs

```diff
@@ -1,30 +1,31 @@
 ﻿using System;
-using System.Web;
+using System.Text.RegularExpressions;
 
-/// <summary>
-/// Summary description for oib
-/// </summary>
-public class oib
+public class Validators
 {
-    public static bool CheckOIB(string oib)
-    {
-        if (oib.Length != 11) return false;
+	public static bool IsValidOIB(string oib)
+	{
+		if (string.IsNullOrEmpty(oib) || !Regex.IsMatch(oib, "^[0-9]{11}$")) 
+			return false;
 
-        long b;
-        if (!long.TryParse(oib, out b)) return false;
+		var a = 10;
+		for (var i = 0; i < 10; i++)
+		{
+			a = a + Convert.ToInt32(oib.Substring(i, 1));

```

Comment:
I added a PR (https://github.com/domagojpa/oib-validation/pull/9), however, since the validation grows overhere :)
I'd consider using Span with Slice instead of Substring.

Reply:
Nice touch! Please make a new PR aligned with the current code.

Reply:
@apozgaj Span/Slice are for high-perf string manipulation, maybe its a overkill for this very simple scenario.

Reply:
I agree that it's not a high-perf string manipulation and oib is not huge string, however, it would be a (micro) optimization. In this case substring returns 10 strings each by 16 bytes while with the Span each char would take only 2 bytes. 

I just suggested - doesn't have to be approved.

Reply:
I like it (and the fact Hudo doesn't is a bonus :-)), it'll be approved, just create new PR.